lib/src/authentication/create/create_account_view.dart

```diff
@@ -0,0 +1,133 @@
+                  ),
+                  InputPassword(
+                    controller: viewModel.passwordController,
+                    onChange: viewModel.onFormChange,
+                    enabled: !viewModel.isBusy,
+                    onEditingComplete: () async =>
+                        viewModel.onEditingCompleted(),
+                  ),
+                  InputPassword(
+                    controller: viewModel.passwordConfirmationController,
+                    passwordToConfirmController: viewModel.passwordController,
+                    onChange: viewModel.onFormChange,
+                    enabled: !viewModel.isBusy,
+                    onEditingComplete: () async =>
+                        viewModel.onEditingCompleted(submit: true),
+                  ),
+                  if (viewModel.busy(viewModel.isCreateAccountButtonEnabled))
+                    const CircularProgressIndicator()
+                  else
+                    SizedBox(
+                      width: double.infinity,
+                      child: Padding(
+                        padding: const EdgeInsets.symmetric(
+                            horizontal: 12.0, vertical: 8.0),
+                        child: FilledButton(
+                          key: const Key(
+                              "create_account_view_create_account_button"),
+                          onPressed: viewModel.isCreateAccountButtonEnabled
+                              ? viewModel.createAccount
+                              : null,
+                          child: Text(
+                            l10n.create_account_view_create_account,
+                            style: const TextStyle(fontSize: 20.0),
+                          ),
+                        ),
+                      ),
+                    ),
+                  AppSpacer.p8(),
+                  Row(
+                    children: [
+                      textDivider,
+                      Text(
+                        l10n.create_account_view_or_separator,
+                        style: const TextStyle(fontSize: 20.0),
+                      ),
+                      textDivider
+                    ],
+                  ),
+                  AppSpacer.p16(),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      ThirdPartyRoundIconButton.google(
+                        onPressed: viewModel.isBusy
+                            ? null
+                            : viewModel.createAccountGoogle,
+                      ),
+                      // Display the Apple button only on iOS
+                      if (defaultTargetPlatform == TargetPlatform.iOS)
+                        ThirdPartyRoundIconButton.apple(
+                          onPressed: viewModel.isBusy
+                              ? null
+                              : viewModel.createAccountApple,
+                        ),
```

Comment:
Missing a `Padding` of `left: 40`, I would prefer using an `AppSpacer` in those cases but Flutter doesn't support multiple children in an if-loop so padding it is!

Reply:
Added